src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
```

Comment:
need to add constructor

Reply:
<img width="514" alt="Screenshot 2023-09-18 at 21 17 03" src="https://github.com/mate-academy/js_herbivores_and_carnivores/assets/123739041/34750fb7-7f9b-4aa2-b956-8a039ce7865a">

when i add the constructor i receive following eslint error

Reply:
when i add the constructor i receive following eslint error
<img width="532" alt="Screenshot 2023-09-18 at 21 21 58" src="https://github.com/mate-academy/js_herbivores_and_carnivores/assets/123739041/5165d33d-d553-4620-a018-7654aac416cd">